lib/assets/javascripts/attachinary.js.coffee

```diff
@@ -12,6 +11,10 @@
             <li>
               <% if(files[i].resource_type == "raw") { %>
                 <div class="raw-file"></div>
+              <% } else if(previewUrl) { %>
```

Comment:
seems like a good workaround to me. Can we make sure that the commit message reflects why we need this? Like `allows previewURL to be passed in for images that were created before transformations were locked down, since they won't have a raw resource type` (or whatever is actually causing this, because I'm not entirely sure :) )